float64 waypoint_x
float64 waypoint_y

